tests/testutils/test_output_line.py

```diff
@@ -56,10 +57,10 @@ def test_output_line_from_message(message: Callable) -> None:
 def test_output_line_to_csv(confidence: Confidence, message: Callable) -> None:
     output_line = OutputLine.from_msg(message(confidence))
     csv = output_line.to_csv()
-    expected_column = "column" if PY38_PLUS else ""
+    expected_column = "2" if PY38_PLUS else "0"
     assert csv == (
         "missing-docstring",
-        "line",
+        "1",
```

Comment:
Note that we are checking the `to_csv()` method here, so `line` and `column` should be `str` again.